ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/recaptcha/v1/recaptcha/_cq_dialog/.content.xml

```diff
@@ -0,0 +1,188 @@
+                                            sling:resourceType="granite/ui/components/coral/foundation/container">
+                                        <items jcr:primaryType="nt:unstructured">
+                                            <required
+                                                    jcr:primaryType="nt:unstructured"
+                                                    sling:resourceType="granite/ui/components/coral/foundation/form/checkbox"
+                                                    checked="{Boolean}true"
+                                                    fieldLabel="Required"
+                                                    sling:hideResource="{Boolean}false"
+                                                    labelledBy="Mandatory"
+                                                    name="./required"
+                                                    granite:class="cmp-adaptiveform-base__required"
+                                                    text="Required"
+                                                    value="true">
+                                            </required>
+                                            <mandatoryMessage
+                                                    jcr:primaryType="nt:unstructured"
+                                                    sling:resourceType="granite/ui/components/coral/foundation/form/textarea"
+                                                    fieldDescription="Error message shown when required field is left empty."
+                                                    granite:class="cmp-adaptiveform-base__mandatorymessage"
+                                                    fieldLabel="Error message"
+                                                    name="./mandatoryMessage"/>
+                                            <validateExpMessage
+                                                    jcr:primaryType="nt:unstructured"
+                                                    sling:resourceType="granite/ui/components/coral/foundation/form/textarea"
+                                                    fieldDescription="Displayed when validation script fails."
+                                                    fieldLabel="Script validation message"
+                                                    name="./validateExpMessage"/>
+                                        </items>
+                                    </column>
+                                </items>
+                            </columns>
+                        </items>
+                    </validation>
+                    <patterns
+                            jcr:primaryType="nt:unstructured"
+                            jcr:title="Patterns"
+                            sling:hideResource="{Boolean}true"
+                            sling:resourceType="granite/ui/components/coral/foundation/container"></patterns>
+                    <help
```

Comment:
if we are hiding help, then Question mark, long description and short description should not be applied. But Readme suggests otherwise.